scripts/content/animenzb.js

```diff
@@ -2,7 +2,7 @@ function addToSABnzbdFromAnimenzb() {
 	var addLink = this;
 	
 	// Set the image to an in-progress image
-	var img = chrome.runtime.getURL('images/sab2_16_fetching.png');
+	var img = chrome.extension.getURL('images/content_icon_fetching.png');
```

Comment:
`chrome.runtime` appears to be the newer method. Why switch to `chrome.extension`?